models/brats_mri_axial_slices_generative_diffusion/configs/train_autoencoder.json

```diff
@@ -0,0 +1,225 @@
+{
+    "imports": [
+        "$import functools",
+        "$import glob",
+        "$import scripts"
+    ],
+    "bundle_root": ".",
+    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "ckpt_dir": "$@bundle_root + '/models'",
+    "tf_dir": "$@bundle_root + '/eval'",
+    "dataset_dir": "@bundle_root",
+    "perceptual_loss_model_weights_path": null,
```

Comment:
please add `pretrained` flag, otherwise it's always `True` for PerceptualLoss

Reply:
Thank you!